litmus-portal/frontend/src/views/CreateWorkflow/WeightSlider/__test__/WeightSlider.test.tsx

```diff
@@ -0,0 +1,40 @@
+import '@testing-library/jest-dom';
+import { screen } from '@testing-library/react';
+import React from 'react';
+import { render } from '../../../../testHelpers/test-util';
+import WeightSlider from '../index';
+
+describe('Weight Slider', () => {
+  it('should have maximum and minimum values', () => {
+    render(
+      <WeightSlider
+        testName="experiment1"
+        weight={8}
+        index={0}
+        handleChange={() => {}}
+      />
+    );
+
+    const slider = screen.getByRole('slider');
+    expect(slider.getAttribute('aria-valuemax')).toBe('10');
+    expect(slider.getAttribute('aria-valuemin')).toBe('0');
```

Comment:
am I not understanding something here...why are we checking aria-values?

Reply:
These aria properties help us identify the min and max values of the slider. When we mock the component we get access to the `mark` function only through the PrettoSlider component thus it renders all the properties it has out of which we find out the ones to test, as far as I know, we can't directly extract `mark` thus the alternative way of checking via properties.

Reply:
I believe there might be some better way...we need to look into it....for now I'm approving the PR